core/src/main/resources/reference.conf

```diff
@@ -327,11 +364,30 @@ base {
         }
       }
       server {
+        remote-address-header = on
         parsing {
           max-content-length = 2048m
           illegal-header-warnings = off
         }
       }
+
+      ## Akka-http session related settings
+      session {
+        clientSession {
+          cookie {
+            name = "gearpump_token"
+            ## domain = "..."
+            path = "/"
+            ## maxAge = 0
+            secure = false
+            httpOnly = true
+          }
+
+          ## Session lifetime.
+          maxAgeSeconds = 3600
+          encryptData = true
+        }
+      }
     }
```

Comment:
It will not.
If security.ui-authentication-enabled is false, then it behave same as 0.7.3